### PR TITLE
FAQ: automatically resolve PD service

### PR DESCRIPTION
**Pull request by l0k0ms**
https://trello.com/c/MNMlxiYI/

### CODE REVIEW

File: content/integrations/pagerduty.md [hunk 1]
```diff
@@ -18,7 +18,18 @@ You can also check out [this documentation][1] from Pagerduty.
 Once you have Pagerduty integrated, you can check out our custom [Pagerduty Incident Trends][2].
 {{< img src="integrations/pagerduty/pagerduty_incident_trends.png" >}}
 
-   [1]: http://www.pagerduty.com/docs/guides/datadog-integration-guide/
-   [2]: https://app.datadoghq.com/report/pagerduty
+## Frequently Asked Questions
+### How to automatically resolve a PagerDuty service when a monitor recovers?
```
Comment (kmshultz) [inline content/integrations/pagerduty.md]: "How do I", not "How to"

File: content/integrations/pagerduty.md [hunk 2]
```diff
@@ -18,7 +18,18 @@ You can also check out [this documentation][1] from Pagerduty.
...
+
+In order to automatically resolve a PagerDuty service when a monitor recovers
```
Comment (kmshultz) [inline content/integrations/pagerduty.md]: Omit "In order"

s/pagerduty/PagerDuty

File: content/integrations/pagerduty.md [hunk 3]
```diff
@@ -18,7 +18,18 @@ You can also check out [this documentation][1] from Pagerduty.
 Once you have Pagerduty integrated, you can check out our custom [Pagerduty Incident Trends][2].
 {{< img src="integrations/pagerduty/pagerduty_incident_trends.png" >}}
 
-   [1]: http://www.pagerduty.com/docs/guides/datadog-integration-guide/
-   [2]: https://app.datadoghq.com/report/pagerduty
+## Frequently Asked Questions
```
Comment (kmshultz) [inline content/integrations/pagerduty.md]: just "FAQ" is fine.

File: content/integrations/pagerduty.md [hunk 4]
```diff
@@ -18,7 +18,18 @@ You can also check out [this documentation][1] from Pagerduty.
...
+### How to automatically resolve a PagerDuty service when a monitor recovers?
+
+In order to automatically resolve a PagerDuty service when a monitor recovers
+you have to include the pagerduty notification in the `{{is_resolve}}` context of your **Say what's happening** section of your monitor as follows:
```
Comment (kmshultz) [inline content/integrations/pagerduty.md]: s/pagerduty/PagerDuty

> you have to

you must

**Comment by l0k0ms** [comment]
https://trello.com/c/ovmBFwoZ/ ^new FAQ

**Comment by kmshultz** [comment]
It looks like some commits from other branches/PRs got mixed into this PR. Can you please close this one out and open a new PR with only the relevant parts?

**Pull request closed without merging**